### DISCUSSION TITLE
display: none equivalent

### DISCUSSION THREAD

**Discussion by saurabh-lohiya** [fastn-stack/fastn#655] [Q&A] [answered]
What is the CSS equivalent of `display: none` in FTD?

**Comment by Arpita-Jaiswal** [comment, marked as answer]
If you want to hide an element in `ftd`, you can use the `if` condition with an expression that evaluates to false.

Here's an example:

```ftd
-- ftd.text: Hello
if: { false }
```

This will prevent the element from being displayed on the page, effectively mimicking the `display: none` behavior in CSS.
